CHANGELOG.md

```diff
@@ -19,29 +186,29 @@ If you have issues upgrading to Babel 7, please double check the [related docume
 
 The [`babel-bridge`](https://github.com/babel/babel-bridge) library may be used if you need to use libraries that have not yet upgraded to Babel 7. You may also enforce the Babel 7 dependency via tools like [yarn resolutions](https://yarnpkg.com/lang/en/docs/selective-version-resolutions/). Overall, you need to ensure all the `@babel/*` deps are fixed at version `7.0.0-beta.47`.
 
-### **Node 8** is now the minimum required version
+#### **Node 8** is now the minimum required version
 
 Trailing commas are now allowed.
 
-### **iOS 9** is now the minimum required version
+#### **iOS 9** is now the minimum required version
 
 Any device that can run iOS 8, can upgrade to iOS 9. Developers who support iOS 8 in their apps may continue doing so as this is a Xcode-level setting (`IPHONEOS_DEPLOYMENT_TARGET`).
 
-### **Xcode 9** is now the minimum required version
+#### **Xcode 9** is now the minimum required version
 
 We recommend using Xcode 9.4 as that is what we use to run our tests.
 
-### **Android** projects are now compiled using the _Android 26 SDK_
+#### **Android** projects are now compiled using the _Android 26 SDK_
```

Comment:
Isn't this 27 ?

Reply:
This was a formatting fix to 0.56, it is not part of 0.57's changes.